### DISCUSSION TITLE
Is there a limitation on how many streams you have ?

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6219] [General]
Is there a limitation on how many streams you have ? in community editions ?
please advice , also sometimes i recieve audio and no video ???

**Comment by burak-58** [comment]
Hi @brixxnatt,
Thank you for your question. Let me convert this issue to a discussion. You will get an answer there.

**Comment by Mohit-3196** [comment]
Hi @brixxnatt,
How are you?

There is no limitation about the number of streams and players either in the community or enterprise edition. It depends upon the instance size and hardware specs. The higher the instance hardware specs like CPU, RAM, bandwidth, etc. the more streams you can have.

Thank you.
